artifactory/commands/buildinfo/xrayscan.go

```diff
@@ -73,8 +67,10 @@ func (bsc *BuildScanCommand) Run() error {
 
 	// Check if should fail build
 	if bsc.failBuild && scanResults.Summary.FailBuild {
-		bsc.buildFailed = true
-		return errorutils.CheckError(errors.New(scanResults.Summary.Message))
```

Comment:
Shouldn't the new error contain the Xray scan results summary message?

Reply:
Since we're using a pre-defined error, we can't set its error message. I think it is okay though, because you get the whole result JSON from Xray printed anyway.